### PR TITLE
Completed javascript-operators-and-expressions - Mohamed 

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-02-26 at 1 53 08 PM

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+var width = 50;
+var height= 10;
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:2]: Please put a space before and after operators

File: javascript-operators-and-expressions/main.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+var area = width * height;
+console.log("value of area:", area);
+console.log("type of area:", typeof area)
+var bill = 10;
+var payment= 20;
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:7]: Please put a space before and after operators

File: javascript-operators-and-expressions/main.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+var change= payment- bill;
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:8]: Please put a space before and after operators

File: javascript-operators-and-expressions/main.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+console.log("value of change:", change);
+console.log("type of change:", typeof change);
+var quizzes = 60;
+var midterm= 60;
+var final= 70;
+var grade= (quizzes+midterm+final) / 3;
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:14]: Please put a space before and after operators

File: javascript-operators-and-expressions/main.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+console.log("value of grade:", grade);
+console.log("type of grade:", typeof grade);
+var firstName = "Mohamed";
+var lastName= "Elmessiry"
+var fullName= firstName+" "+ lastName;
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:19]: Please put a space before and after operators

File: javascript-operators-and-expressions/main.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+console.log("value of fullName:", fullName);
+console.log("type of fullname:", typeof fullName);
+var pH = 10;
+var isAcidic= (pH < 7);
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:23]: Please put a space before and after operators

File: javascript-operators-and-expressions/main.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+console.log("value of isAcidic:",isAcidic);
+console.log("type of isAcidic:", typeof isAcidic);
+var headCount = 50;
+var isSparta= headCount === 300;
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:27]: Please put a space before and after operators

File: javascript-operators-and-expressions/main.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+console.log("value of isSparta:", isSparta);
+console.log("type of isSparta:", typeof isSparta);
+var motto = fullName +" "+"is the GOAT";
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:30]: Please put a space before and after operators

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Please look again at your code _**very closely**_, there are still many operators without a space before and after them

File: javascript-operators-and-expressions/main.js (hunk 8 above)
Comment (Mohamed-Elmessiry) [reply javascript-operators-and-expressions/main.js:30]: It's weird, as I pushed the exercise when it's spaced out, will push again :)

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**